src/logging.rs

```diff
@@ -202,9 +203,11 @@ impl OnDemandLogFile {
                 Ok(file) => {
                     self.file = Some(io::LineWriter::new(file));
                     self.created.store(true, Ordering::Relaxed);
+                    // TODO: Fix the format here...
                     let _ = writeln!(io::stdout(), "Created log file at {:?}", self.path);
                 }
                 Err(e) => {
+                    // TODO: Fix the format here...
                     let _ = writeln!(io::stdout(), "Unable to create log file: {}", e);
```

Comment:
Fixing these might require a bit of thought if we want them to be formatted consistently with the various loggers that may be used.

Reply:
How so? It seems like they're already pretty much formatted like the other messages.

Reply:
I'm talking about the formatting applied by the various loggers. Sorry should have been more specific.